packages/commons-test/src/containerTestUtils.ts

```diff
@@ -17,7 +17,7 @@ export const TEST_MINIO_IMAGE =
 
 export const TEST_MAILPIT_HTTP_PORT = 8025;
 export const TEST_MAILPIT_SMTP_PORT = 465;
-export const TEST_MAILPIT_IMAGE = "axllent/mailpit";
+export const TEST_MAILPIT_IMAGE = "axllent/mailpit:v1.19";
```

Comment:
Unrelated, for consistency with https://github.com/pagopa/interop-be-monorepo/pull/725/files#r1672464296